From 71a58613493dd0211a576ed6531c5ee712a03ff3 Mon Sep 17 00:00:00 2001
From: Eric Bower <me@erock.io>
Date: Sun, 14 Jul 2024 07:14:44 -0400
Subject: [PATCH 0/2] Add torch deps

I took the liberty of adding a requirements file for python.

Bob Sour (1):
  chore: add torch to requirements

Eric Bower (1):
  feat: lets build an rnn

 README.md        | 4 ++--
 requirements.txt | 1 +
 train.py         | 5 +++++
 3 files changed, 8 insertions(+), 2 deletions(-)
 create mode 100644 requirements.txt
 create mode 100644 train.py

-- 
2.45.2

From 59456574a0bfee9f71c91c13046173c820152346 Mon Sep 17 00:00:00 2001
From: Eric Bower <me@erock.io>
Date: Wed, 3 Jul 2024 15:18:47 -0400
Subject: [PATCH 1/2] feat: lets build an rnn

---
 README.md | 4 ++--
 train.py  | 2 ++
 2 files changed, 4 insertions(+), 2 deletions(-)
 create mode 100644 train.py

diff --git a/README.md b/README.md
index 586bc0d..8f3a780 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,3 @@
-# test
+# Let's build an RNN
 
-testing git pr
+This repo demonstrates building an RNN using `pytorch`
diff --git a/train.py b/train.py
new file mode 100644
index 0000000..5c027f4
--- /dev/null
+++ b/train.py
@@ -0,0 +1,2 @@
+if __name__ == "__main__":
+    print("train!")
-- 
2.45.2


From 71a58613493dd0211a576ed6531c5ee712a03ff3 Mon Sep 17 00:00:00 2001
From: Bob Sour <bob@bower.sh>
Date: Fri, 5 Jul 2024 18:57:03 +0000
Subject: [PATCH 2/2] chore: add torch to requirements

---
 requirements.txt | 1 +
 train.py         | 3 +++
 2 files changed, 4 insertions(+)
 create mode 100644 requirements.txt

diff --git a/requirements.txt b/requirements.txt
new file mode 100644
index 0000000..4968a39
--- /dev/null
+++ b/requirements.txt
@@ -0,0 +1 @@
+torch==2.3.1
diff --git a/train.py b/train.py
index 5c027f4..8f32fe2 100644
--- a/train.py
+++ b/train.py
@@ -1,2 +1,5 @@
+import torch
+
 if __name__ == "__main__":
     print("train!")
+    print(torch.rand(6,6))
-- 
2.45.2

